entities/build.gradle

```diff
@@ -3,6 +3,7 @@ plugins {
 }
 dependencies {
     implementation(project(":common"))
+    implementation(project(":authz"))
```

Comment:
you can remove it.

Reply:
this is due to dependency on ResourceType which is used in RoleBindingNode entity

Reply:
this is due to dependency on ResourceType which is used in RoleBindingNode entity